### PR TITLE
#9 README.md updated

### PR DESCRIPTION
**Pull request by vanekbr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,72 +1,55 @@
...
-Set: Search for tslint.json.
+1. If you haven't done it already, [make a fork of this repo](https://github.com/o360/frontend/fork).
+1. Clone to your local computer using `git`.
+1. Make sure that you have Node 12 or later installed. See instructions [here](https://nodejs.org/en/download/).
+1. Make sure that you have `yarn` installed; see instructions [here](https://yarnpkg.com/lang/en/docs/install/).
+1. Run `yarn` (no arguments) from the root of your clone of this project to install dependencies.
 
-## Install packages:
-  `yarn install`
+## Environment Configuration
```
Comment (yaroslavdudchenko) [inline README.md]: IMO this section should be reworked entirely.

Actual config files should be removed and config example should be added instead. And finally example file should be described in the section, option by option.

**Comment by yaroslavdudchenko** [comment]
Please, add Run in Docker section, as it is the main implied deployment scenario.

File: README.md (hunk 1 above)
Comment (vanekbr) [reply README.md]: You're right, it will be done in #44

**Comment by vanekbr** [comment]
> Please, add Run in Docker section, as it is the main implied deployment scenario.

@yaroslavdudchenko It will be updated in #45

**Pull request merged**